### ISSUE TITLE
issues of training on sceneflow

### ISSUE THREAD

**Issue by Chenwei-Huang** [NVlabs/FoundationStereo#144]
Great job!  I'd like to know if there are any specific considerations when training solely on the SceneFlow dataset. I trained on SceneFlow with the same settings as selective-IGEV. However, NaN appeared in the training loss during the descent process.

**Comment by Xiaochui8** [comment]
Hello. Did you solve this problem? I tried to fine-tune the released model on a subset of FSD dataset(about 60k image pairs). After about 2 epochs, the model start to output NaN on some samples, and the ratio of NaN samples increased gradually. 
Did you train from scratch or fine-tune on the released model?
I also tried to train from scratch on the same data, however, it didn't appear NaN.

**Comment by NOCNOA** [comment]
> Hello. Did you solve this problem? I tried to fine-tune the released model on a subset of FSD dataset(about 60k image pairs). After about 2 epochs, the model start to output NaN on some samples, and the ratio of NaN samples increased gradually. Did you train from scratch or fine-tune on the released model? I also tried to train from scratch on the same data, however, it didn't appear NaN.

hi，i met same question,did you solve it?

**Comment by Xiaochui8** [comment]
> > Hello. Did you solve this problem? I tried to fine-tune the released model on a subset of FSD dataset(about 60k image pairs). After about 2 epochs, the model start to output NaN on some samples, and the ratio of NaN samples increased gradually. Did you train from scratch or fine-tune on the released model? I also tried to train from scratch on the same data, however, it didn't appear NaN.
> 
> hi，i met same question,did you solve it?

You can use this open-source training code(https://github.com/XiandaGuo/OpenStereo.git).

But it can't totally solve this problem. It seems like that there are some bugs in FoundationStereo. For example, in extractor.py, ContextNetDino.conv2.bn has very very high variance, which probably cause the NaN during inference. I find that it's hard to finetune FoundationStereo.
